buildout.cfg

```diff
@@ -1,21 +1,22 @@
 [buildout]
-develop =
+develop = . ../zodb
```

Comment:
``../zodb`` is temporary for my convenience, as I'm evolving documentation in 2 repos at the same time.

I really should combine them. :)